deploy/role.yaml

```diff
@@ -25,6 +25,7 @@ rules:
   verbs:
   - 'get'
   - 'list'
+  - 'watch'
```

Comment:
good catch! It is related to the log entry we spoke about yesterday?

Reply:
Yes